### PR TITLE
initial browser tests for badges T88795

### PR DESCRIPTION
**Pull request by WMDE-Fisch**

### CODE REVIEW

File: tests/browser/features/step_definitions/badges_steps.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Given(/^I have at least (\d+) badges to test$/) do |num|
+  @exisitng_badges = on(ItemPage).existing_badges
```
Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/badges_steps.rb]: typo in exisitng_badges

File: tests/browser/features/step_definitions/badges_steps.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  expect(@exisitng_badges.length).to be >= num.to_i
+end
+
+When /^I click the empty badge selector$/ do
+  on(ItemPage).sitelinks_form[1].empty_badge_element.when_visible.click
+end
```
Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/badges_steps.rb]: This line (and most of this file) is not indented properly.

Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/badges_steps.rb]: Is RuboCop running on TravisCI? How come it did not catch this? https://github.com/wmde/WikidataBrowserTests/blob/master/.travis.yml#L7

Comment (WMDE-Fisch) [inline tests/browser/features/step_definitions/badges_steps.rb]: the indenting is all the same in this project
what should it be like? maybe rubocop is not configured correctly?

File: tests/browser/features/sitelinks_badge.feature [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      And Sitelink empty badge selector should be there
+
+  @ui_only
+  Scenario: Sitelink badge UI shows all available badges
+    When I click the sitelink edit button
+      And I type en into the 1. siteid input field
+      And I click the empty badge selector
+    Then Sitelink pagename input field should be there
+      And Sitelink badge selector menu should be there
+      And Sitelink badge selector menu should be show available badges
```
Comment (thiemowmde) [inline tests/browser/features/sitelinks_badge.feature]: "should be show" is no proper English, I think.

**Comment by thiemowmde** [comment]
+1, except for the grammar error I found, see above.

File: tests/browser/features/step_definitions/badges_steps.rb (hunk 2 above)
Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/badges_steps.rb]: Sorry, I was reviewing the code on a small 11 inch screen, and looks like github does not display diffs correctly in some cases, indenting looked completely broken.

**Pull request merged**